matchzoo/processor_units/processor_units.py

```diff
@@ -531,3 +531,64 @@ def transform(self, tokens: list) -> list:
                              'pad mode.'.format(self._pad_mode))
 
         return fixed_tokens.tolist()
+
+
+class MatchingHistogramUnit(ProcessorUnit):
+    """
+    MatchingHistogramUnit Class.
+
+    :param hist_bin_size: The number of bins of the matching histogram.
+    :param embedding_matrix: The word embedding matrix applied to calculate
+                             the matching histogram.
+    :param normalize: Boolean, normalize the embedding or not.
+    :param mode: The type of the historgram, it should be one of 'CH', 'NG',
+                 or 'LCH'.
+
+    Examples:
+        >>> embedding_matrix = np.array([[1.0, -1.0], [1.0, 2.0], [1.0, 3.0]])
+        >>> text_left = [0, 1]
+        >>> text_right = [1, 2]
+        >>> histogram = MatchingHistogramUnit(3, embedding_matrix, True, 'CH')
+        >>> histogram.transform([text_left, text_right])
+        [[3.0, 1.0, 1.0], [1.0, 2.0, 2.0]]
+
+    """
+
+    def __init__(self, hist_bin_size: int = 30, embedding_matrix=None,
+                 normalize=True, mode: str = 'LCH'):
+        """The constructor."""
+        self._hist_bin_size = hist_bin_size
+        self._embedding_matrix = embedding_matrix
+        if normalize:
+            self._embedding_matrix = self.normalize_embedding(embedding_matrix)
```

Comment:
`normalize_mebedding` is a classmethod, can you use `self`? not sure why use classmethod.